src/app/components/Filters/FieldsetDate.jsx

```diff
@@ -34,9 +34,9 @@ class FieldsetDate extends React.Component {
     const value = e.target.value;
     let displayValue = '';
 
-    if (e.target.name === 'start-date') {
+    if (e.target.name === 'dateAfter') {
```

Comment:
Is there a point in doing this check? Why not `displayValue = e.target.name;`? You do have a check later on for `if (displayValue)`. You can also update that check to be `if (displayValue && displayValue !== 'dateAfter' ...` etc.